molli/pipeline/crest.py

```diff
@@ -64,3 +64,62 @@ def conformer_search(self, output: JobOutput, mol: ml.Molecule, *args, **kwargs)
             blank_conf.coords = conf_geom.coords
 
         return result
+
+    @Job().prep
+    def conformer_screen(
+        self,
+        ens: ml.ConformerEnsemble,
+        charge: int = None,
+        mult: int = None,
+        method: str = None,
+        ewin: float = None,
+        mdlen: float = None,
+        mddump: float = None,
+        vbdump: float = None,
+        chk_topo: bool = None,
+        misc: str = None,
+    ):
+        #  -{method} -ewin {ewin:0.4f} -mdlen {mdlen:0.4f} -mddump {mddump:0.4f} -vbdump {vbdump:0.4f} -T {self.nprocs}"""
+        cmd = f"""{self.executable} -screen input.xyz -T {self.nprocs}"""
+
+        if method:
+            cmd += f" -{method}"
+
+        if charge is None:
+            charge = ens.charge
+
+        if mult is None:
+            mult = ens.mult
+
+        cmd += f" -chrg {charge} -uhf {mult - 1}"
+
+        for var in "ewin", "mdlen", "mddump", "vbdump":
+            if (val := locals()[var]) is not None:
+                cmd += f" -{var} {val:0.4f}"
+
+        if chk_topo:
+            if not chk_topo:
+                cmd += " --noreftopo"
+
+        if misc is not None:
+            cmd += f" {misc}"
+
+        return JobInput(
+            ens.name,
+            commands=[(cmd, "crest")],
+            files={"input.xyz": ens.dump_xyz().encode()},
+            return_files=["crest_ensemble.xyz"],
+        )
+
+    @conformer_screen.post
+    def conformer_screen(self, output: JobOutput, mol: ml.Molecule, *args, **kwargs):
+        all_geoms = ml.CartesianGeometry.loads_all_xyz(
+            output.files["crest_ensemble.xyz"].decode()
+        )
+
+        result = ml.ConformerEnsemble(mol, n_conformers=len(all_geoms))
+
+        for blank_conf, conf_geom in zip(result, all_geoms):
+            blank_conf.coords = conf_geom.coords
```

Comment:
Can this be limited to a list of Cartesian Geometries? i.e.

`result = ml.ConformerEnsemble(mol, n_conformers=len(all_geoms), coords=all_geoms)`

or even something such as

`result = ml.ConformerEnsemble(mol, all_geoms)`

I only ask because I know we can load from a list of molecules.